fedimg/services/ec2/ec2initiate.py

```diff
@@ -158,10 +158,16 @@ def main(image_urls, access_id, secret_key, regions, volume_types=None,
                 published_images.extend(publisher.publish_images(
                     region_image_mapping=[(region, image.id)]
                 ))
+        except ValueError:
+            _log.debug('Source not found.')
+            return []
         except Exception as e:
             _log.debug(e.message)
+            return []
```

Comment:
so here we are returning `[]` where we weren't sending anything back before. wehere is this main function getting called from?